doc/nvim-tree-lua.txt

```diff
@@ -1277,6 +1302,10 @@ exists.
     - navigate.prev
     - navigate.select
 
+- api.async:                                            *nvim-tree.api.async*
```

Comment:
This is correlated with `events.subscribe_async` to make user able to interfere with async execution flow of file operations. My initial intention here is to supplement https://github.com/nvim-tree/nvim-tree.lua/pull/1821 to be able to use async version of lsp request instead of blocking [`request_sync`](https://github.com/antosha417/nvim-lsp-file-operations/blob/2d2d3466a7a92be86a596fc9751809825e60be64/lua/lsp-file-operations/will-rename.lua#L18). It's fine to leave it for next PR.